src/app/ApplicationTemplate.Shared.Views/Views/Content/Authentication/CreateAccountPage.xaml

```diff
@@ -0,0 +1,302 @@
+﻿<Page x:Class="ApplicationTemplate.Views.Content.CreateAccountPage"
```

Comment:
I don't understand how this file (and all the other ones following) is added? Shouldn't it be only moved (and not **both moved and added**)?

Reply:
I'm not 100% sure, I can try something

Reply:
I thought maybe it was because I cut-pasted, but even when I drag, for some reason, it just does that.

Reply:
- src/app/ApplicationTemplate.Shared.Views/**Views**/Content/Authentication/CreateAccountPage.xaml
- src/app/ApplicationTemplate.Shared.Views/Content/Authentication/CreateAccountPage.xaml

Reply:
also the folder structure is weird and repeats itself. You should remove duplicates.